cwe_checker_rs/src/checkers/cwe_78/context/mod.rs

```diff
@@ -121,8 +124,8 @@ impl<'a> Context<'a> {
         let source = self.taint_source.unwrap();
         let name = self.taint_source_name.clone().unwrap();
         let description: String = format!(
-            "(Potential OS Command Injection) {} ({}) -> {}",
-            sub_name, source.tid.address, name
+            "(Input for call {} is not properly sanitized) {} ({}) -> {}",
+            name, sub_name, source.tid.address, name
```

Comment:
I didn't mean that the `(Potential OS Command Injection)` should be removed. More like
```
(Potential OS Command Injection) Input for call to {} is not properly sanitized in function {} ({})
```
i.e. combine the two. :-) This would also be the same style as the other CWE warnings.